### ISSUE TITLE
Best network utils library

### ISSUE THREAD

**Issue by jesusprubio** [assaultjs/assaultjs-utils-network#6]
For now I've added this library: https://github.com/rs/node-netmask

But, there're another two deeply used:
- https://github.com/indutny/node-ip
- https://github.com/binarysec/node-ipaddr

Should we select only one and contribute to it? Should we have ours one?

Moreover I've exposed the full Netmask object to avoid rewriting wrappers for any function we need and be able to use the original documentation (KISS).

Opinions?

**Comment by vertexclique** [comment]
`node-ip` has something like that
```
ip.not('255.255.255.0') // 0.0.0.255
ip.or('192.168.1.134', '0.0.0.255') // 192.168.1.255
ip.isPrivate('127.0.0.1') // true
```

but `node-netmask` doesn't have these (from my sight). I think we should summarize all modules in our one (if everybody wants the same). And `node-netmask` doesn't have v6, does it?

**Comment by jesusprubio** [comment]
Yep, I agree. BTW node-ip IPv6 support isn't enough, but it's something :).
